src/policy/satisfy.rs

```diff
@@ -71,7 +71,7 @@ impl<Pk: MiniscriptKey + ToPublicKey> Policy<Pk> {
                 }
             }
             Policy::Older(n) => {
-                if satisfier.check_older(Sequence(*n)) {
+                if satisfier.check_older(Sequence(*n as u32)) {
```

Comment:
In d975984dd4b6db1fc27ae55c1a175c4d6964167b:

If this is going from a u16 to a u32 you should use `.into()` rather than casting.